common/changes/@rushstack/heft-typescript-plugin/fix-typescript-error_2023-07-11-21-40.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft-typescript-plugin",
+      "comment": "Only make warnings terminal if \"buildProjectReferences\" is true.",
```

Comment:
What does "make warnings terminal" mean?

Reply:
Do you mean "make warnings a hard error?" Using the word "terminal" makes it sound like you're conditionally writing to a console.

Reply:
"terminal" as in "stops the build"